spec/javascripts/health-care/components/error-message.spec.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ReactTestUtils from 'react-addons-test-utils';
+import ErrorMessage from '../../../../_health-care/_js/_components/error-message';
+import _ from 'lodash';
+
+describe('<ErrorMessage>', () => {
+  let component = null;
+
+  beforeEach(() => {
+    component = ReactTestUtils.renderIntoDocument(
+      <ErrorMessage message="There is an error here"/>
+    );
+    assert.ok(component, "Cannot even render component");
+  });
+
+  it('has sane looking features', () => {
+    const inputs = ReactTestUtils.scryRenderedDOMComponentsWithTag(component, 'span');
+    expect(inputs).to.have.length(1);
+  });
+
+  it('has the appropriate className', () => {
```

Comment:
Not quite sure if this is just a clarifying comment or a question, but your test case sounds completely right to me.

In something this simple (no dependencies, etc), you could just write a "Shallow Render" test against a golden-result w/o having any sort of fragility. See the `Shallow rendering` section here:
  https://facebook.github.io/react/docs/test-utils.html

This test case can just be something like:
```
var renderer = ReactTestUtils.createRenderer();
result = renderer.getRenderOutput();
expect(result.type).toBe('div');
expect(result.props.children).to.eql([
   <span className="usa-input-error-message" role="alert">There is an error here</span>
]);
```

Notice the .to.eql() instead of .to.equal() above. Jest (the test framework used in React) uses toEqual() which according to their docs (https://facebook.github.io/jest/docs/api.html) is a deep equality check. In mocha, .to.equal() is shallow. Stupid JS and the million of repeated frameworks. grr.

If you don't like the golden value test pattern and prefer the more specific method you have currently, then we need another `it` test to ensure that the message attribute renders into the span.